### PR TITLE
Add more precise information to README

### PR DESCRIPTION
**Pull request by Inventitech**
Hey @ClintonCao,

could you check this PR and add the XML to generate the reports, replacing some snippet?
Thx!

Moritz

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,11 +67,21 @@ SIG Feedback: [SIG Feedback](https://github.com/ClintonCao/Contextproject-TSE/bl
...
 
 ## Running the Program:
-* Go to the `src/main/java/BlueTurtle/TSE` folder.
-* Run the file `Main.java`.
+* Add the following snippets to the project's pom.xml file that you want to analyze
+```XML
+Some snippet
+```
+
+* Run `mvn site` on this project (this generates the ASAT reports which are read-in by UAV)
```
Comment (ClintonCao) [inline README.md]: Using this might be a bit misleading to the user (as they might run `mvn site` on UAV instead?).

**Comment by Inventitech** [comment]
Thx for the comment, addressed it.

**Comment by ClintonCao** [comment]
XML snippet is added.

**Comment by Inventitech** [comment]
I think it's quite OK now.

**Comment by ClintonCao** [comment]
I'll merge this then

**Pull request merged**